packages/features/dub/ReferralClient.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+
+import { DubEmbed } from "@dub/embed-react";
+import { useTheme } from "next-themes";
+
+import { IS_DUB_REFERRALS_ENABLED } from "@calcom/lib/constants";
+
+const ReferralClientImpl = ({ publicToken }: { publicToken: string }) => {
```

Comment:
- if this is used in apps/web/app only, it can exist there. better be in `refer` dir
- that also means `@dub/embed-react` can only be in `apps/web/package.json`